scala3doc/src/dotty/renderers/MemberRenderer.scala

```diff
@@ -10,7 +10,7 @@ import collection.JavaConverters._
 
 class MemberRenderer(signatureRenderer: SignatureRenderer, buildNode: ContentNode => String)(using DocContext):
 
-  private val converter = new DocTagToContentConverter()
+  private val converter = ScalaCommentToContentConverter
```

Comment:
Shouldn't this be injected?

Reply:
This is a code that intend to not dependend on dokka as much as possible hence no injection